pkg/apis/policy/v1alpha1/propagation_types.go

```diff
@@ -197,6 +197,13 @@ type ClusterAffinity struct {
 	// ExcludedClusters is the list of clusters to be ignored.
 	// +optional
 	ExcludeClusters []string `json:"exclude,omitempty"`
+
+	// PropagatePriority describes propagating preference priorities under the specified constraints.
+	// Clusters are descending sorted by weight and scheduler will try to propagate to the preferred
+	// clusters under the premise of constraints, e.g., SpreadConstraints, AvailableReplicas, etc.
+	// Note that the clusters who have more than one priority will only take effect with the greater one.
+	// +optional
+	PropagatePriority []corev1.PreferredSchedulingTerm `json:"propagatePriority,omitempty"`
```

Comment:
How do you think the name `PreferredDuringScheduling`(short for `PreferredDuringSchedulingIgnoredDuringExecution`)?  Or `PreferredClusters`?